src/word.c

```diff
@@ -0,0 +1,67 @@
+#include "dictionary.h"
+#include <stdio.h>
+#include <strings.h>
+#include <string.h>
+#include <assert.h>
+#include <stdlib.h>	
+
+int num_punctuation = 22;
+char* punctuation_array[] = {"+",",",":"," ",".","-","'","&","!","?",":",";","#","~","=","/","$","£","^","\n","_","<",">"};
```

Comment:
Also, no globals declared like this. That's just bad practice.

Reply:
left inside is_in_array